django-backend/fecfiler/f3x_summaries/models.py

```diff
@@ -387,9 +387,24 @@ class ReportMixin(models.Model):
 
     def save(self, *args, **kwargs):
         if self.report:
-            self.report.calculation_status = None
-            self.report.save()
-            logger.info(f"F3X Summary: {self.report.id} marked for recalcuation")
+            committee = self.report.committee_account
+            report_date = self.report.coverage_from_date
+            if report_date is not None:
+                report_year = report_date.year
+
+                reports_to_flag_for_recalculation = F3XSummary.objects.filter(
+                    committee_account=committee,
+                    coverage_from_date__year=report_year,
+                    coverage_from_date__gte=report_date,
+                    upload_submission=None
```

Comment:
I think we'll need to be a bit more specific.  Definitely 'REJECTED' and probably 'PENDING' uploads will still need to calcuate their dsps.
`
                    ~models.Q(upload_submission__fec_status=models.Value("ACCEPTED")),`

Reply:
:+1: